app/[localeCode]/(story)/secret/[uuid]/page.tsx

```diff
@@ -12,6 +13,7 @@ interface Props {
         localeCode: Locale.Code;
         uuid: StoryRef['uuid']; // story secret_uuid
     };
+    searchParams: Record<string, string>;
```

Comment:
I wonder why `searchParams` is generic here, and is more specific in the definition above.

Reply:
I think I've made it more specific when I was first implementing it and then I realised that it doesn't really matter to the "route" to be too specific here, because we pass the `searchParams` to the parsing function anyway, which checks and picks the keys it expects.

So I think I can use the generic type everywhere.